common/display/displayplanemanager.cpp

```diff
@@ -111,6 +111,9 @@ std::tuple<bool, DisplayPlaneStateList> DisplayPlaneManager::ValidateLayers(
   auto layer_begin = layers.begin();
   auto layer_end = layers.end();
   bool render_layers = false;
+  // Below flag takes care of condition,when the cursor layer is assigned the
+  // lower zorder
+  bool cursor_plane_enable = true;
```

Comment:
Atually the other bug here is the for loop for (auto j = layers.rbegin(); j != layers.rend(); ++j) 

it should be for (auto j = layer_begin; j != layer_end; ++j)